airflow/www/extensions/init_views.py

```diff
@@ -39,7 +40,7 @@
 log = logging.getLogger(__name__)
 
 # airflow/www/extensions/init_views.py => airflow/
-ROOT_APP_DIR = path.abspath(path.join(path.dirname(__file__), path.pardir, path.pardir))
+ROOT_APP_DIR = Path(__file__).resolve().parents[2]
```

Comment:
```suggestion
ROOT_APP_DIR = Path(__file__).parents[2].resolve()
```

There’s a difference if any parent is a symlink. This better matches the original semantic.